### PR TITLE
composer require should not have the flag --no-update

### PR DESCRIPTION
**Pull request by OskarStark**
Replaces #957 

### Subject

> composer require should not have the flag --no-update when only one package is required.
> 
> Would introduce some confusion

### CODE REVIEW

**Comment by OskarStark** [comment]
Thank you @core23 👍

File: Resources/doc/reference/installation.rst [hunk 1]
```diff
@@ -25,7 +25,7 @@ Retrieve the bundle with composer:
 
 .. code-block:: bash
 
-    $ php composer.phar require sonata-project/media-bundle --no-update
+    $ php composer.phar require sonata-project/media-bundle
```
Comment (Soullivaneuh) [inline Resources/doc/reference/installation.rst]: On a separated commit, could you please simplify this?

```bash
composer require sonata-project/media-bundle
```

Comment (Soullivaneuh) [inline Resources/doc/reference/installation.rst]: Erf, didn't see it's already merged.

Comment (OskarStark) [inline Resources/doc/reference/installation.rst]: will do

Comment (OskarStark) [inline Resources/doc/reference/installation.rst]: done in #1050

**Pull request merged**